<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cocos2d-x的android神器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>……</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pathShowTextView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathShowLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prePathShowLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>../../Classes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://blog.54gamer.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;元谷的博客&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
